tutorials/frontend/deploy_sparse.py

```diff
@@ -119,6 +117,9 @@
 # determines how sparse the generated weights should be. The higher
 # the sparsity, the faster the result.
 sparsity = 0.85
+# Running benchmarking mode might overload CI,
+# so it is disabled by default.
+benchmark = False
```

Comment:
Could you say something like "Switch this to true to run the benchmark yourself"?

Reply:
Done!

Reply:
I think this should be a more explicit to the user. I can see people skipping over this. How about "Change this to to true to run the benchmark." Also, could you move it to the end of the script so it makes logical sense to someone reading it.

Reply:
@tkonolige : I think moving to end may not look good, as already there are many settings defined at one place in the top section. 
And about the message i had changed to "If true, then all relay models(dense and sparse) will be benchmarked."
Please let me know, if this is not clear. TIA!

Reply:
All the other tutorials put it at the bottom, so we should be consistent. It also makes sense for the reader. The last step they want to do is run everything.

Reply:
Okay agree with your point, thanks for clarification! Just to confirm we should move all the config or only benchmark config to bottom?

Reply:
Just benchmark

Reply:
Done!